hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java

```diff
@@ -2946,33 +2946,8 @@ public ListKeysLightResult listKeysLight(String volumeName,
   public List<RepeatedOmKeyInfo> listTrash(String volumeName,
       String bucketName, String startKeyName, String keyPrefix, int maxKeys)
       throws IOException {
-    boolean auditSuccess = true;
```

Comment:
Same as the previous comments (for the other portions of code as well)

Reply:
Updated in the latest commit [b645e54](https://github.com/apache/ozone/pull/7060/commits/b645e5448bf06a866dfff844e640a6bd64053bf9)